conf/luna.conf

```diff
@@ -63,7 +63,7 @@ HomeDoubleClickDuration=150
 
 [Memory]
 CardLimit=-1
-AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging
+AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging;org.webosports.app.phone;org.webosports.app.contacts;org.webosports.app.messaging
```

Comment:
Let's drop them now, no need to keep them in that list, even if they are installed from 3.0.x.